testsrc/org/mozilla/javascript/tests/scriptengine/BuiltinsTest.java

```diff
@@ -31,8 +31,24 @@ public void setup() {
     }
 
     @Test
-    public void printStdout() throws ScriptException {
```

Comment:
@nmondal can you please bring back to old test and add you as a new one

Reply:
@rbri  the old test was not testing anything.
If the old test was "correct" we would never have had the bug in the first place.
Being said that, if you still wish we can. But do we want to?

Reply:
I guess the test at least makes sure the print function is supported and works without an exception.
And it does not harm to have one more test (and it is good for the statistics :-))

Reply:
Sure! Will do it.

Reply:
> I guess the test at least makes sure the print function is supported and works without an exception. And it does not harm to have one more test (and it is good for the statistics :-))

Done. Done. Done.
🕺🏽 

@rbri can  you please do the honors of running the pipeline?